tests/foreman/cli/test_gpgkey.py

```diff
@@ -462,7 +462,7 @@ def test_positive_update_key_for_empty_product(self):
         product = Product.info({'id': product['id'], 'organization-id': self.org['id']})
         self.assertEqual(product['gpg']['gpg-key'], gpg_key['name'])
         # Update the gpg key
-        new_name = gen_choice(valid_data_list().values())
+        new_name = gen_choice(list(valid_data_list().values()))
```

Comment:
Just a improvement suggestion:

```suggestion
        new_name = gen_choice([*valid_data_list().values()])
```

Reply:
I'd actually argue its better to explicitly use list here, since we're essentially type casting.